templates/commands/render/flags.go

```diff
@@ -123,6 +127,14 @@ func (r *RenderFlags) Register(set *cli.FlagSet) {
 		Usage: "(experimental) write a manifest file containing metadata that will allow future template upgrades.",
 	})
 
+	f.StringVar(&cli.StringVar{
+		Name:    "upgrade-channel",
+		Target:  &r.UpgradeChannel,
+		Default: "",
```

Comment:
Curious What does default "" means in `upgrade-channel`?

Reply:
That's explain in the usage text below: `The default is to upgrade from the branch that the template was originally rendered from if rendered from a branch, or in any other case to use the value "latest" to upgrade to the latest release tag by semver order.`